core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/description_fr.properties

```
@@ -0,0 +1,2 @@
+blurb = Informe les administrateurs à propos de l''osbolescence d''un ou plusieurs plugins installés. \
+  Les sites de mise à jour fournissent ces données, la documentation du plugin indique généralement pourquoi le plugin est obsolète.
```

```suggestion
  Cette information est fournise par les sites de mise à jour, la documentation du plugin indique généralement pourquoi le plugin est obsolète.
```
More close to the original sentence, subjective & optional.

---

Just a typo in suggestion: `est fournie`

---

D'oh, good catch! I've edited my suggestion 🙂